hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
I've been doing lots of ANSI C programming recently so block comments are drilled into me.